dev-lua/luafilesystem/luafilesystem-1.6.3.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A Lua library for file system operations"
+DESCRIPTION="
+LuaFileSystem is a Lua library developed to complement the set of functions \
+related to file systems offered by the standard Lua distribution.
+LuaFileSystem offers a portable way to access the underlying directory \
+structure and file attributes."
+HOMEPAGE="http://keplerproject.github.io/luafilesystem/"
+COPYRIGHT="2004-2012 Kepler Project"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/keplerproject/luafilesystem/archive/v_1_6_3.tar.gz"
+CHECKSUM_SHA256="5525d2b8ec7774865629a6a29c2f94cb0f7e6787987bf54cd37e011bfb642068"
+SOURCE_DIR="luafilesystem-v_1_6_3"
+
+#check for Lua version, we need to remove the dot for the headers
+LUAVERSION_WITH_DOTS=$(lua -v | sed 's/Lua //' | head -c3)
+LUAVERSION_WITHOUT_DOTS=$(echo $LUAVERSION_WITH_DOTS | tr -d '.')
```

Comment:
Been doing some checks yesterday (still wip) where I renamed LUAVERSION_WITH_DOTS to LUAVERSION  (it gives the dot default) and LUAVERSION_WITHOUT_DOT (as there is only one dot in there) ...